tavern/internal/secrets/gcp_test.go

```diff
@@ -0,0 +1,37 @@
+package secrets_test
+
+// No way to test this in CI but worked in dev
```

Comment:
Sorry i'm leaving dead code but I think it's nice to have as a way to test manually.